trl/extras/vllm_client.py

```diff
@@ -142,6 +142,7 @@ def generate(
         min_p: float = 0.0,
         max_tokens: int = 16,
         guided_decoding_regex: Optional[str] = None,
+        stop: Optional[list[str]] = None,
```

Comment:
Do I understand correctly that this is needed to enable e.g. stopping generation at the end of a code block instead of EOS?